inst/www/htmlwidgets.js

```diff
@@ -584,109 +716,14 @@
           return;
         el.className = el.className + " html-widget-static-bound";
 
```

Comment:
So what happened to the localstorage code below?

Reply:
It has moved into HtmlwidgetContainer.ts.  All storage of state is now done in one place.